gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/util/EmitFiles.java

```diff
@@ -40,7 +42,11 @@ public static Path replaceVars(InputStream src, Path dest, Map<String, String> v
         dest.getParent().toFile().mkdirs();
 
         // write content
-        return Files.write(dest, text.getBytes(StandardCharsets.UTF_8));
+        try {
```

Comment:
I was thinking, what if we relaxed the "doesn't throw exception" constraint? if we were writing tasks normally, with the `@TaskAction` we could just slap a `throws Exception` on the end and everything would be fine. In the generated task, we just wrap invocations with a try catch? thoughts?

Reply:
only because it just makes code a bit bloated and distracting / or you have to make a million "unsafe write" type methods

Reply:
I think the only downside would be another layer of rethrown exception in the stacktrace that does not have any useful information.

Reply:
I guess the RuntimeException I've made below is not really that useful either. But the autowrapper exception would be something like "Task LaunchConfigTask failed"